<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseFacetDialog</class>
 <widget class="QDialog" name="ChooseFacetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facet</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../risorse.qrc">
    <normaloff>:/icon/images/icon.png</normaloff>:/icon/images/icon.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QGroupBox" name="typeGroup">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="Enumeration">
        <property name="text">
         <string>enumeration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MinExclusive">
        <property name="text">
         <string>minExclusive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MinInclusive">
        <property name="text">
         <string>minInclusive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MaxExclusive">
        <property name="text">
         <string>maxExclusive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MaxInclusive">
        <property name="text">
         <string>maxInclusive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="TotalDigits">
        <property name="text">
         <string>totalDigits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="FractionDigits">
        <property name="text">
         <string>fractionDigits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Length">
        <property name="text">
         <string>length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MinLength">
        <property name="text">
         <string>minLength</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="MaxLength">
        <property name="text">
         <string>maxLength</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="WhiteSpace">
        <property name="text">
         <string>whiteSpace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Pattern">
        <property name="text">
         <string>pattern</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Value:</string>
         </property>
         <property name="buddy">
          <cstring>value</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="value"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="lblFixed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Fixed:</string>
         </property>
         <property name="buddy">
          <cstring>fixed</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fixed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>&lt;Not used&gt;</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>value</tabstop>
  <tabstop>fixed</tabstop>
  <tabstop>Enumeration</tabstop>
  <tabstop>MinExclusive</tabstop>
  <tabstop>MinInclusive</tabstop>
  <tabstop>MaxExclusive</tabstop>
  <tabstop>MaxInclusive</tabstop>
  <tabstop>TotalDigits</tabstop>
  <tabstop>FractionDigits</tabstop>
  <tabstop>Length</tabstop>
  <tabstop>MinLength</tabstop>
  <tabstop>MaxLength</tabstop>
  <tabstop>WhiteSpace</tabstop>
  <tabstop>Pattern</tabstop>
 </tabstops>
 <resources>
  <include location="../../risorse.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChooseFacetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChooseFacetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
